Added documentation for the project globs (#385)
spacelift-io/user-documentation: docs/concepts/stack/stack-settings.md

!!! info
    The project root can be overridden by the [runtime configuration](../configuration/runtime-configuration/README.md#project_root-setting) specified in the `.spacelift/config.yml` file.

### Project globs

> Project globs allow you to specify files and directories outside of the project root that the stack cares about. In the absence of push policies, any changes made to the project root and any paths specified by project globs will trigger Spacelift runs.

Reviewer: ```suggestion
Project globs option allows you to specify files and directories outside of the project root that the stack cares about. In the absence of push policies, any changes made to the project root and any paths specified by project globs will trigger Spacelift runs.
```

![](../../assets/screenshots/Stack_Settings_Project_Globs.png)

You aren't required to add any project globs if you don't want to, but you have the option to add as many project globs as you want for a stack.

> Under the hood, project globs take advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match) function to do pattern matching.

Reviewer 2: ```suggestion
Under the hood, project globs takes advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match) function to do pattern matching.
```

Author: Under the hood, the project globs feature takes advantage

> Under the hood, project globs take advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match) function to do pattern matching.

Reviewer: ```suggestion
Under the hood, project globs take advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match){: rel="nofollow"} function to do pattern matching.
```

### Project globs

The project globs option allows you to specify files and directories outside of the project root that the stack cares about. In the absence of push policies, any changes made to the project root and any paths specified by project globs will trigger Spacelift runs.

![](../../assets/screenshots/Stack_Settings_Project_Globs.png)

You aren't required to add any project globs if you don't want to, but you have the option to add as many project globs as you want for a stack.

> Under the hood, the project globs feature takes advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match){: rel="nofollow"} function to do pattern matching.

Reviewer: feature or option? I think we should be consistent on these terms.

Author: Let's stick with option.
